docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+<a id="create-store-example"></a>
+
+```
+    var messageStore = Stores.createStore({
+      _messages: {},
+      getAll: function () {
+        return Object.keys(this._messages);
+      }
+    });
+```
+
+Below is the API documentation for this method.
+
+###<a id="Stores-createStore"></a>Stores.createStore [#](#Stores-createStore)
+
+```
+    var store = Stores.createStore(methods);
+```
+
+####<a id="methods"></a>Parameter - `methods` - type: OBJECT - optional [#](#methods)
+Methods and properties to be extended on the store to be created. This object is optional and there are no expected keys.
+
+####<a id="storeReturn"></a>Return - `store` - type: OBJECT [#](#storeReturn)
+The `Stores.createStore` method returns an object which is described below.
+
+###<a id="store-emitChange"></a>Method `store.emitChange` - type: FUNCTION [#](#store-emitChange)
+A store instance is able to invoke the `emitChange` method, which will emit a change event string for any listeners that are listening for that particular string. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.emitChange(); // will emit the default change event type string: 'CHANGE'
+```
+
+#####<a id="emitChange-CHANGE_EVENT"></a>Parameter - `CHANGE_EVENT` - type: STRING - optional [#](#emitChange-CHANGE_EVENT)
+Change event type string that will be broadcast for any listeners that are listening for that event type. If no `CHANGE_EVENT` is specified the default value of `CHANGE` will be used.
+
+```
+    var newChangeEvent = 'NEW_CHANGE_EVENT';
+    store.emitChange(newChangeEvent);
+```
+
+###<a id="store-addChangeListener"></a>Method `store.addChangeListener` - type: FUNCTION [#](#store-addChangeListener)
+A store instance is able to invoke the `addChangeListener` method, which will invoke the provided callback function when the specified event type string is emitted. The change event can be either the default event string `CHANGE` or one provided by the user.
+
+```
+    store.addChangeListener(callback, CHANGE_EVENT);
+```
+
+#####<a id="addChangeListener-callback"></a>Parameter - `callback` - type: FUNCTION - required [#](#addChangeListener-callback)
+The callback function that will be called when the change event type string that is being listened for with this invocation of addChangeListener is broadcast.
```

Comment:
Change sentence to:  The callback to invoke when the corresponding event is triggered.